### PR TITLE
chore: update to reference main branch after default branch migration

### LINKED ISSUE #7178: Update references to Bigtable in Authoring Guide
The example directory for Bigtable referenced in the authoring guide no longer exists in this repository. Since it's just an example it should be renamed to another appropriate product.

### PR DESCRIPTION
**Pull request by dandhlee**
## Description

Python-docs-samples repo now uses `main` branch as default branch, updating references to point to the new main branch.

Kokoro config updates in #7167.

Towards #7153.
Fixes #7178.

## Checklist
- [x] Checked all links to ensure they're well formatted and valid.
- [x] Checked for any missing references. Any references left over will be resolved in a separate PR or is out of scope for this migration (third party repositories).
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [x] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: MAC_SETUP.md [hunk 1]
```diff
@@ -172,7 +172,7 @@ created virtual environment. You can install this by running
     pyenv will make your virtualenv the default Python. Since you specified	
     more than one version, it will also add binaries like `python36` and	
     `python27` to your PATH, which
-    [nox](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#using-nox)
+    [nox](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#using-nox)
```
Comment (dandhlee) [inline MAC_SETUP.md:175]: link needs to be updated

File: AUTHORING_GUIDE.md [hunk 2]
```diff
@@ -81,20 +81,20 @@ for an example.
 that corresponds to the Google Cloud service or API used by the sample.
 For example, a sample demonstrating how to work with BigTable should be
 in a subfolder under the
-[python-docs-samples/bigtable](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/bigtable)
+[python-docs-samples/bigtable](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/bigtable)
```
Comment (dandhlee) [inline AUTHORING_GUIDE.md:84]: All references to python-docs-samples/main/bigtable should be updated

File: data-science-onramp/feature-engineering/README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 ### [Citibike trips](https://console.cloud.google.com/bigquery?p=bigquery-public-data&d=new_york_citibike&t=citibike_trips)
 * BigQuery public dataset
-* Output of [data cleaning code](https://github.com/Symmetries/python-docs-samples/tree/master/data-science-onramp/data-cleaning)
```
Comment (dandhlee) [inline data-science-onramp/feature-engineering/README.md:7]: data cleaning has been renamed to data processing: see https://github.com/GoogleCloudPlatform/python-docs-samples/pull/5770

Comment (dandhlee) [reply data-science-onramp/feature-engineering/README.md:7]: Rename to https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/data-science-onramp/data-processing

**Review by m-strzelczyk — APPROVED** [review]

**Review by kurtisvg — APPROVED** [review]

**Review by leahecole — APPROVED** [review]

**Pull request merged**